### PR TITLE
Build ppc64le image

### PR DESCRIPTION
**Pull request by mayurwaghmode**

### CODE REVIEW

**Review by bcrochet — CHANGES_REQUESTED** [review]
Please squash these commits to a single commit.

**Comment by mayurwaghmode** [comment]
@bcrochet I have squashed commits. Please review!

File: Dockerfile-ppc64le [hunk 1]
```diff
@@ -0,0 +1,44 @@
+ARG quay_expiration=never
+ARG release_tag=0.0.0
+
+# golang:1.16 image created 2021-06-24T00:31:06.02014601Z
+FROM docker.io/golang AS builder
```
Comment (bcrochet) [inline Dockerfile-ppc64le:5]: Please add an explicit tag for the version. If 1.17 is available for ppc, use that.

File: Dockerfile-ppc64le [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+ARG quay_expiration
+ARG release_tag
+
+# Build the preflight binary
+COPY . /go/src/preflight
+WORKDIR /go/src/preflight
+RUN make build RELEASE_TAG=${release_tag}
+
+# ubi8:latest
+FROM registry.access.redhat.com/ubi8/ubi:latest
```
Comment (bcrochet) [inline Dockerfile-ppc64le:15]: Is there no ubi-minimal for ppc?

File: Dockerfile-ppc64le [hunk 3]
```diff
@@ -0,0 +1,44 @@
+ARG quay_expiration=never
```
Comment (bcrochet) [inline Dockerfile-ppc64le:1]: Small nit: I'd prefer the filename to be Dockerfile.ppc64le instead of with the '-' but not a blocker.

**Review by bcrochet — CHANGES_REQUESTED** [review]

File: Dockerfile-ppc64le (hunk 2 above)
Comment (mayurwaghmode) [reply Dockerfile-ppc64le]: Yes, it is available.

File: Dockerfile.ppc64le [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+      gzip \
+      iptables \
+      findutils \
+    && dnf clean all
+
+# Install OpenShift client binary
+RUN curl -L https://mirror.openshift.com/pub/openshift-v4/clients/ocp/${OPENSHIFT_CLIENT_VERSION}/openshift-client-linux-${OPENSHIFT_CLIENT_VERSION}.tar.gz | tar -xzv -C /usr/local/bin oc
+
+# Install Operator SDK binray
+RUN curl -Lo /usr/local/bin/operator-sdk https://github.com/operator-framework/operator-sdk/releases/download/v${OPERATOR_SDK_VERSION}/operator-sdk_linux_ppc64le \
```
Comment (bcrochet) [inline Dockerfile.ppc64le:40]: Couldn't we just make an ARCH ARG? Default it to amd64, and override it in the action? That way we don't need an entirely new Dockerfile. And i'm fine with bumping the go version in the main dockerfile.

**Review by bcrochet — CHANGES_REQUESTED** [review]
Sorry to keep coming back with more changes. I think I'll be satisfied after this. :)

File: Dockerfile.ppc64le (hunk 4 above)
Comment (mayurwaghmode) [reply Dockerfile.ppc64le]: @bcrochet I have made the changes. Please review it!

File: Dockerfile [hunk 5]
```diff
@@ -2,9 +2,10 @@ ARG quay_expiration=never
 ARG release_tag=0.0.0
```
Comment (bcrochet) [inline Dockerfile:2]: You can add the arg here with the default (=amd64). Then you don't even have to change build-main.yaml.

**Review by bcrochet — CHANGES_REQUESTED** [review]

**Review by bcrochet — APPROVED** [review]
/lgtm

**Comment by mayurwaghmode** [comment]
@bcrochet can you assign the label again? I wanted to test with the latest changes, so did a few commits. Done with the testing.

**Review by acornett21 — APPROVED** [review]

**Review by bcrochet — APPROVED** [review]
/lgtm

**Pull request merged**